### PR TITLE
Add signers and finality to block status api.

### PR DESCRIPTION
**Pull request by dinhln89**

### CODE REVIEW

**Comment by thanhson1085** [comment]
lgtm. @ngtuna take a look

File: core/blockchain.go [hunk 1]
```diff
@@ -1572,3 +1575,18 @@ func (bc *BlockChain) SubscribeChainSideEvent(ch chan<- ChainSideEvent) event.Su
...
 	return bc.scope.Track(bc.logsFeed.Subscribe(ch))
 }
+
+// Get current IPC Client.
+func (bc *BlockChain) GetClient() (*ethclient.Client, error) {
+	if bc.Client == nil {
+		// Inject ipc client global instance.
+		client, err := ethclient.Dial(bc.IPCEndpoint)
+		if err != nil {
+			log.Error("Fail to connect RPC", "error", err)
```
Comment (ngtuna) [inline core/blockchain.go]: RPC -> IPC ?

**Review by ngtuna — APPROVED** [review]
lgtm

**Comment by ngtuna** [comment]
Some more changes I just suggested to Dinh. Will hold this a bit

**Comment by ngtuna** [comment]
cool thanks @dinhln89 . Merging now

**Pull request merged**